### PR TITLE
Tutorial

### PR DESCRIPTION
**Pull request by stbrewer85**
Reupload of tutorial

### CODE REVIEW

File: docs/0.1/getting-started/tutorial.md [hunk 1]
```diff
@@ -1,7 +1,104 @@
...
+## Make a Rule 
+
+A rule governs the data you have uploaded in a specific recType. You can adjust who can see the data and what they can do with it. In this section we will allow a user of your choosing to read and write to the data you have posted to microshare.
+
+1. Navigate to [our platform](../file)(https://dapp.microshare.io)
+2. Click the "Manage" button in the top toolbar
+3. Click the "Rules" button in the left toolbar
+4. Click the the "Create" button
+5. Input the Rule name in the top most text field
+5. Type "My First Rule" in the text field labeled "Description"
```
Comment (jgaye) [inline docs/0.1/getting-started/tutorial.md]: Numbers here and below are stuck at 5

File: docs/0.1/getting-started/tutorial.md [hunk 2]
```diff
@@ -1,7 +1,104 @@
...
+
+You have now made a Rule to govern access to your data!     
+
+## Create a Robot 
+
+1. Navigate to [our platform][../file](https://dapp.microshare.io)
+2. Click the "Manage" button in the top toolbar
+3. Click the "Robots" button in the left toolbar
+4. Click the "Create" button 
+4. Input the Robot name in the top most text field
```
Comment (jgaye) [inline docs/0.1/getting-started/tutorial.md]: Numbers here and below are stuck at 4

File: docs/0.1/getting-started/tutorial.md [hunk 3]
```diff
@@ -1,7 +1,104 @@
...
+3. Post data to Microshare  
+4. Make a rule to govern this data  
+5. Create a robot to moniter this data  
+6. Make a dashboard to view this data  
+
+This process will require Postman. It is avaliable to download and install [here](https://www.getpostman.com/)
+
+## Post Data to Microshare
+
+Make a post call using the same parameters as before, but with two notable additions.
```
Comment (jgaye) [inline docs/0.1/getting-started/tutorial.md]: At least add a link to the API page

File: docs/0.1/getting-started/tutorial.md [hunk 4]
```diff
@@ -1,7 +1,104 @@
...
+Make a post call using the same parameters as before, but with two notable additions. 
+
+1. Include the data you wish to upload into the body of the call in JSON format. For example:
+
+    {"Test":"Data"}
+    
+2. Include a header with a key of "recType" 
+    * This will create a space for the data and attribute the above value as the recType's name  
+        * It is recommended that your first recType follow a simple naming convention I.E. "yourFirstName.yourLastName"
+
```
Comment (jgaye) [inline docs/0.1/getting-started/tutorial.md]: A screenshot from postman here would have been best. Check the assets, there should be an image you can use.

File: docs/0.1/getting-started/tutorial.md [hunk 5]
```diff
@@ -1,7 +1,104 @@
...
+
+## Make a Rule 
+
+A rule governs the data you have uploaded in a specific recType. You can adjust who can see the data and what they can do with it. In this section we will allow a user of your choosing to read and write to the data you have posted to microshare.
```
Comment (jgaye) [inline docs/0.1/getting-started/tutorial.md]: Add a link to the Share page

File: docs/0.1/getting-started/tutorial.md [hunk 6]
```diff
@@ -1,7 +1,104 @@
...
+5. Click on the "user" dropdown within the "Requestor" section
+5. Select the "Specific Value" option
+    * A text box will appear with greyed out text reading "Please enter a value"
+5. Enter the email of the user whom will be allowed to access the data
+    * An example of a valid input would be "example@microshare.io"
+    * Note: this new user's email must associated with an account on our platform in order to access the data
+
+* Click the "Create" button at the top of the page 
+
+You have now made a Rule to govern access to your data!
```
Comment (jgaye) [inline docs/0.1/getting-started/tutorial.md]: Good but a couple of screenshots might help with understanding

File: docs/0.1/getting-started/tutorial.md [hunk 7]
```diff
@@ -1,7 +1,104 @@
...
+    * A text box will appear with greyed out text reading "Please enter a value"
+5. Enter the email of the user whom will be allowed to access the data
+    * An example of a valid input would be "example@microshare.io"
+    * Note: this new user's email must associated with an account on our platform in order to access the data
+
+* Click the "Create" button at the top of the page 
+
+You have now made a Rule to govern access to your data!     
+
+## Create a Robot
```
Comment (jgaye) [inline docs/0.1/getting-started/tutorial.md]: Add a Link to the Robot guide

File: docs/0.1/getting-started/tutorial.md [hunk 8]
```diff
@@ -1,7 +1,104 @@
...
+    var TO = 'INPUT YOUR EMAIL HERE';
+    var SUBJECT = 'This is a test email';
+    var BODY = 'This is a test email!';
+
+    lib.sendMicroshareEmail(TO, SUBJECT, BODY);
+
+}
+
+{% endhighlight %}
+
```
Comment (jgaye) [inline docs/0.1/getting-started/tutorial.md]: Another screenshot here could help. Again, there are some in the assets you might be able to use directly.

**Comment by jgaye** [comment]
Please have a look at the comments @deemcv @stbrewer85 
I think this tutorial can get much better with not much work.

**Comment by deemcv** [comment]
@steve, I also realize that you never explain how to "create an account" or
"get an access token" are these mentioned somewhere else in the tutorial or
on the site? If they are previously mentioned I think we should link out to
that, if not, a quick blurb and maybe a screenshot could help.

**Comment by jgaye** [comment]
And take any content yo uwant from there:
http://docs.microshare.io/docs/0.1/advanced/lorawan-devices/hackiot_philly/ but
please do not link to that page, it's only for the hackathon

File: docs/0.1/getting-started/tutorial.md (hunk 1 above)
Comment (stbrewer85) [reply docs/0.1/getting-started/tutorial.md]: It's ok - the numbers are like placeholders in markup. When viewed from the site it looks like a 1,2,3,4,5,6,7,8,9 - not just 5 over and over again.

**Comment by stbrewer85** [comment]
Hey so I had explained how to create an account and get an access token - but moved it to another section. It has since been lost. I am adding to the edits I am currently making.

**Pull request merged**